src/game_objects/minigame/penguin.cpp

```diff
@@ -0,0 +1,77 @@
+#include "penguin.h"
+#include "asset_loader.h"
+#include "events.h"
+
+Penguin::Penguin(int id) : GameObject(id) {
+    tag = "PENGUIN";
+
+    if (id == ON_SERVER) {
```

Comment:
Shouldn't move speed and spawn location be the responsibility of the spawner (e.g. `Scene`)? We can just send them in as parameters to the constructors.